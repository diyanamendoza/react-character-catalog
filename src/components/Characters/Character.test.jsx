import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Character from './Character';

it('should display the provided character', () => {
  const { container } = render(
    <MemoryRouter>
      <Character
        character={{
          image: 'https://placebear.com/200/300',
          name: 'Ted',
          species: 'bear',
          status: 'Alive',
        }}
      />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
  screen.getByText(/Ted/i)
});
