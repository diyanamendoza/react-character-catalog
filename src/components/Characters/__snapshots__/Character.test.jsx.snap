// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display the provided character 1`] = `
<div>
  <figure>
    <img
      alt="Ted"
      src="https://placebear.com/200/300"
    />
    <figcaption>
      <h3>
        Ted
      </h3>
      <p>
        bear
      </p>
      <p>
        Status: 
        Alive
      </p>
    </figcaption>
  </figure>
</div>
`;
